packages/mobile/src/app/reducers.ts

```diff
@@ -47,6 +52,11 @@ const initialState = {
   hideVerification: false,
   showRaiseDailyLimitTarget: undefined,
   walletConnectEnabled: false,
+  rewardsABTestThreshold: '0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF',
```

Comment:
Since we use accounts in lowercase I think we need this to be `0xffffff....ffff`, otherwise all addresses starting with letters (`0xa`+) will belong to the B group